podpac/core/coordinates/array_coordinates1d.py

```diff
@@ -347,14 +347,21 @@ def _select(self, bounds, return_indices, outer):
             try:
                 gt = self.coordinates >= max(self.coordinates[self.coordinates <= bounds[0]])
             except ValueError as e:
-                gt = self.coordinates >= -np.inf
+                if self.dtype == np.datetime64:
+                    gt = ~np.isnat(self.coordinates)
+                else:
+                    gt = self.coordinates >= -np.inf
             try:
                 lt = self.coordinates <= min(self.coordinates[self.coordinates >= bounds[1]])
             except ValueError as e:
-                lt = self.coordinates <= np.inf
```

Comment:
Looks like some kind of special case bug you ran into?

Reply:
yes for certain versions of np, it can't compare dates with `np.inf`